app/models/group/sektions_mitglieder.rb

```diff
@@ -9,15 +9,32 @@ class Group::SektionsMitglieder < ::Group
 
   self.static_name = true
 
+  validates :type, uniqueness: { scope: :parent_id }
+
   ### ROLES
   class Mitglied < ::Role
+    require_relative '../sac_cas/concerns/role_beitragskategorie'
```

Comment:
Geht das nicht ohne `require_relative`? In praktisch allen anderen Wagons haben wir auch (Domain-)Klassen die nur im Wagon existieren und vom Resolver trotzdem gefunden werden. Möglicherweise hättest du beim Entwickeln einfach mal den Rails Prozess neustarten müssen..?

Reply:
ich versuche es nochmals ohne ... es hatte bei den tests immer zu Fehlern geführt da er die role_beitragskategorie class nicht finden konnte